src/js/review/ArrayField.jsx

```diff
@@ -187,7 +187,9 @@ class ArrayField extends React.Component {
             <h5 className="form-review-panel-page-header">{title}</h5>
             {itemsNeeded && <span className="schemaform-review-array-warning-icon"/>}
             {!itemCountLocked &&
-              <button type="button" className="edit-btn primary-outline" onClick={() => this.handleAdd()}>Add Another</button>
+              <button type="button" className="edit-btn primary-outline" onClick={() => this.handleAdd()}>
+                {uiOptions.itemName ? `Add Another ${uiOptions.itemName}` : 'Add Another'}
```

Comment:
Sounds good- it does render the space in the markup but I guess browsers just ignore it.

Reply:
Without the conditional, this renders `undefined` if there's not an `itemName` on `uiOptions`. I tried this with concatenation and sting templates with the same result.

Reply:
Thanks for pointing out the capitalization stuff. Looks like [this](https://github.com/department-of-veterans-affairs/vets-website/blob/6107c6dccacccfcbf8034cf0a97930e38a31784d/src/applications/disability-benefits/all-claims/pages/paymentInformation.js#L21) the only place we're not using consistent capitalization and it's using a custom view instead of the `ArrayField`. 

I will test and communicate this change to the group once this PR is approved but before I merge it.